integration-tests/ahm/Cargo.toml

```diff
@@ -9,44 +9,48 @@ publish = false
 
 [dev-dependencies]
 codec = { workspace = true, default-features = true }
-asset-hub-polkadot-runtime = { workspace = true }
+asset-hub-polkadot-runtime = { workspace = true, default-features = true, features = ["runtime-benchmarks"]}
```

Comment:
You cannot always add it, it will not be correct since we also need to run tests without it.

Reply:
fixed